src/libraries/sendtohelix.proj

```diff
@@ -4,6 +4,9 @@
     <HelixBuild Condition="'$(HelixBuild)' == ''">$(BUILD_BUILDNUMBER)</HelixBuild>
     <HelixBuild Condition="'$(HelixBuild)' == ''">default</HelixBuild>
 
+    <!-- To enable the interpreter for mono, we need to pass an env switch -->
+    <MonoEnvOptions Condition="'$(MonoEnvOptions)' == '' and '$(MonoEnableInterpreter)' == 'true'">--interpreter</MonoEnvOptions>
+
```

Comment:
The Helix `TestRunNamePrefix` should also contain `interpreter` to identify the run in the azure devops test explorer.

Reply:
I put it in the yml.  Seemed to be the best spot.